settingsManager/SettingsManager.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, appName='default', user=None):
 		self.customSettings = {}
 
 		self.rootDir = cOS.ensureEndingSlash(
-			os.environ.get('ARK_CONFIG'))
+			os.environ.get('ARK_ROOT') + '/ark/config')
 
 		# set the file we're trying to load
 		self.filename = self.getFilename(appName)
```

Comment:
this should be a custom file, the rest will happen automatically

appName = arkUtil.makeWebSafe(appName)
self.rootDir + appName + '.json'